understanding/roles.md

```diff
@@ -43,7 +43,7 @@ if(message.member.roles.has(role.id)) {
 
 ```javascript
 // Check if they have one of many roles
-if(message.member.roles.some(r=>["Dev", "Mod", "Server Staff", "Proficient"].includes(r.name)) ) {
+if([...message.member.roles.cache.values()].some(r=>["Dev", "Mod", "Server Staff", "Proficient"].includes(r.name)) ) {
```

Comment:
the destructuring of values should not be necessary. message.member.roles.cache.some() is perfectly valid. Please update, and I can merge this.

Reply:
You're completely right. Updated!